contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -47,12 +47,24 @@
 class DownloadGitRepoAction(Action):
     def __init__(self, config=None, action_service=None):
         super(DownloadGitRepoAction, self).__init__(config=config, action_service=action_service)
+        self.https_proxy = self.config.get('https_proxy', None)
+        self.http_proxy = self.config.get('http_proxy', None)
+        self.ca_bundle_path = self.config.get('ca_bundle_path', None)
```

Comment:
Talking about the `ca_bundle_path`, we can support it as ENV variable `proxy_ca_bundle_path` or similar.